flow/elastic.go

```diff
@@ -138,8 +138,18 @@ func getCommitCallback() (func(int64, []elastic.BulkableRequest), func(int64, []
 		diff := float64(time.Now().Sub(start).Nanoseconds()) / float64(1000000000)
 		prome.ObserveBulkProcessTime(diff)
 
-		for _, response := range response.Items {
-			for _, responseItem := range response {
+		if err != nil {
+			log.Errorf("Bulk commit error: %s", err)
+			return
+		}
+
+		if response == nil {
+			log.Error("Bulk commit error: response is nil")
```

Comment:
https://thanos.io/v0.13/coding-style-guide.md/#rules-for-log-messages

```suggestion
			log.Error("bulk commit error: response is nil")
```